src/CMakeLists.txt

```diff
@@ -13,42 +13,40 @@ add_subdirectory(capture)
 
 add_library(sqlhook SHARED sqlhook.c)
 target_compile_options(sqlhook PRIVATE -fPIC)
-target_include_directories(sqlhook PRIVATE ${LIBSQLITE_INCLUDE_DIR})
-target_link_libraries(sqlhook PRIVATE domain os SQLite::sqlite)
+target_link_libraries(sqlhook PRIVATE domain os SQLite::SQLite3)
```

Comment:
I've removed all of the `target_include_directories(... LIBSQLITE_INCLUDE_DIR)`, since they get automatically added when we use `target_link_libraries(SQLite::SQLite3)`